cli/restart.go

```diff
@@ -26,6 +26,7 @@ func (r *RootCmd) restart() *serpent.Command {
 		Short:       "Restart a workspace",
 		Middleware: serpent.Chain(
 			serpent.RequireNArgs(1),
+			PrintDeprecatedOptions(),
```

Comment:
Good idea to use middleware 👍 
I'm curious why this isn't applied to all CLI actions, though?

Serpent may not have middleware support but we could centralize the definition of the common middlewares; i.e. (almost?) every CLI action needs to call `r.InitClient(client)` and will need this middleware, so let's define them centrally.